support-lib/jni/djinni_support.cpp

```diff
@@ -371,6 +371,15 @@ jstring jniStringFromUTF8(JNIEnv * env, const std::string & str) {
     return res;
 }
 
+jstring jniStringFromWString(JNIEnv * env, const std::wstring & str) {
+
+    std::u16string utf16(str.begin(), str.end());;
```

Comment:
I guess @konovalov-aleks wants to ensure that it is using `char16_t` (std::u16string) so that it can be converted to `unsigned 16 bits` (jchar)? But `std::wstring` is using `wchar_t`, which can be either 16 bits or 32 bits depending on the compiler. This conversion `std::u16string utf16(str.begin(), str.end())` is not safe though.